src/test/java/com/wikia/webdriver/common/core/helpers/UserWithEmailFactory.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import lombok.Value;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'lombok.Value'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)